examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -140,8 +138,10 @@ sed -i "s#<CMD>#export PYTHONPATH=/project/s1053/install/serialbox2_master/gnu/p
 # execute on a gpu node
 sbatch -W -C gpu run.daint.slurm
 wait
-if [ -n "$target_dir" ] ; then
-    rsync *.json $target_dir
+if grep -q SUCCESS slurm-*.out; then
+    echo "performance run sucessful"
+else
+    echo "ERROR: performance run not sucessful"
+    exit 1
 fi
-
-echo "performance run sucessful"
+mv -f slurm-*.out compile.daint.out
```

Comment:
You've be overwriting the output from the compilation step, did you mean for this to be run.daint.out?

Reply:
Good catch.